### DISCUSSION TITLE
Better images for README.md

### DISCUSSION THREAD

**Discussion by danielmarschall** [rustdesk/rustdesk#7398] [General]
Hello,

I have edited the pictures you have in your README.md and censored them better. The black hand-drawn censoring looked a bit odd :-)  I did not censor the LAN address because it is nothing secret.

![Connection Manager](https://github.com/rustdesk/rustdesk/assets/28412477/db82d4e7-c4bc-4823-8e6f-6af7eadf7651)

![Connected to a Windows PC](https://github.com/rustdesk/rustdesk/assets/28412477/9baa91e9-3362-4d06-aa1a-7518edcbd7ea)

![File Transfer](https://github.com/rustdesk/rustdesk/assets/28412477/39511ad3-aa9a-4f8c-8947-1cce286a46ad)

![TCP Tunneling](https://github.com/rustdesk/rustdesk/assets/28412477/78e8708f-e87e-4570-8373-1360033ea6c5)

**Comment by rustdesk** [comment]
Thanks, you can edit directly and submit PR.

**Reply by xlionjuan** [reply]
@rustdesk maybe all screenshot need to replaced by flutter version?

**Reply by rustdesk** [reply]
Unnecessary
